VsVim.sln

```diff
@@ -30,7 +30,7 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Documentation", "Documentat
 		Documentation\Supported Features.md = Documentation\Supported Features.md
 	EndProjectSection
 EndProject
-Project("{6EC3EE1D-3C4E-46DD-8F32-0CC8E7565705}") = "VimCore", "Src\VimCore\VimCore.fsproj", "{333D15E0-96F8-4B87-8B03-467220EED275}"
+Project("{F2A71F9B-5D33-465A-A702-920D77279786}") = "VimCore", "Src\VimCore\VimCore.fsproj", "{333D15E0-96F8-4B87-8B03-467220EED275}"

```

Comment:
Not sure why VSMac changed the project type guid here. I probably should just revert this change.

Reply:
Yeah unsure why it did this. SLN files are mostly a mystery to me though.

Reply:
They are a mystery to everyone I think. Only the machines can understand them :)